graphql_compiler/tests/test_compiler.py

```diff
@@ -443,6 +463,16 @@ def test_filter_on_optional_variable_name_or_alias(self):
                 }}
                 RETURN $matches
             )
+            WHERE
+                (
```

Comment:
Elsewhere, the first `(` is on the same line as the keyword that starts it. Let's keep the formatting like that: `WHERE (` then de-indent everything below by one.